common/src/View/CompilationRunner.cpp

```diff
@@ -266,7 +270,13 @@ namespace TrenchBroom {
             bindEvents(m_currentTask->get());
 
             emit compilationStarted();
-            *m_context << "#### Using working directory '" << m_context->variableValue(CompilationVariableNames::WORK_DIR_PATH) << "'\n";
+
+            const std::string workDir = m_context->variableValue(CompilationVariableNames::WORK_DIR_PATH);
+            if (!QDir(QString::fromStdString(workDir)).exists()) {
+                *m_context << "#### Non-existent working directory '" << workDir << "'\n";
```

Comment:
Shouldn't this message read something like "Error: working directory ... does not exist"?

Reply:
Yeah, fixed.